docs/guides/site_styles/index.md

```diff
@@ -5,3 +5,40 @@ has_children: true
 parent: Guides
 nav_order: 3
 ---
+
+Site Styles is where creators and developers can define colours, fonts and custom CSS.
+
+## Colours 
+The default colours can be defined in this section through a colour picker and later in a dropdown throughout the pages' blocks. These colours are:
```

Comment:
_The default colours can be defined in this section through a colour picker and later in a dropdown throughout the pages' blocks. These colours are:_

I like this but I would just be slightly more explicit, e.g.: 

Four palette colours are defined here. These colours are used as default throughout the theme, and will also be available as 'Palette colours' in any colour picker when editing the site.

Reply:
Ok, much better, thanks!